dictionary.txt

```diff
@@ -56,6 +60,8 @@ BycatchFleets
 Bzero
```

Comment:
I would consider revising all instances of Bzero in the documentation to $B_{0}$

Reply:
I agree for the most part (and have replaced it in all but 2 instances) but I think there are a couple of places it references input and output where SS3 actually has the text as Dynamic Bzero so those 2 instances I will keep since it's referring to the output report 59 Dynamic Bzero and the control file where it has `0 # (0/1/2) read specs for more stddev reporting: 0 = skip, 1 = read specs for reporting stdev for selectivity, size, and numbers, 2 = add options for M,Dyn. Bzero, SmryBio`